src/components/PageElements/Elements/ConsolidateReviewDecision.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ApplicationResponse, ReviewResponse } from '../../../utils/generated/graphql'
+import ApplicantResponseElement from './ApplicantResponseElement'
+import ReviewResponseElement from './ReviewResponseElement'
+import strings from '../../../utils/constants'
+import { Icon } from 'semantic-ui-react'
+
+interface ConsolidateReviewDecisionProps {
+  applicationResponse: ApplicationResponse
+  summaryViewProps: SummaryViewWrapperProps
+  isActiveReviewResponse: boolean
+  isNewApplicationResponse: boolean
+  /* can add check for isNewReviewResponse */
+  reviewResponse?: ReviewResponse
+  originalReviewResponse?: ReviewResponse
+  showModal: () => void
+}
+const ConsolidateReviewDecision: React.FC<ConsolidateReviewDecisionProps> = ({
+  applicationResponse,
+  summaryViewProps,
+  isActiveReviewResponse,
+  isNewApplicationResponse,
+  reviewResponse,
+  originalReviewResponse,
+  showModal,
+}) => {
+  const isConsolidation = true
+  const decisionExists = !!reviewResponse?.decision
+
+  // After consolidation is submitted, reviewResponses are trimmed if they are not changed duplicates
+  // or if they are null, we only want to show reviewResponses that are linked to latestOriginalReviewResponse - from reviewer
+  const isNewConsolidation = originalReviewResponse?.id !== reviewResponse?.reviewResponseLinkId
+
+  return (
+    <div>
+      {/* Application Response */}
+      <ApplicantResponseElement
+        applicationResponse={applicationResponse}
+        summaryViewProps={summaryViewProps}
+      />
+      {/* Consolidation Response */}
+      {!isNewConsolidation && reviewResponse && (
+        <ReviewResponseElement
+          isCurrentReview={true}
+          isConsolidation={isConsolidation}
+          applicationResponse={applicationResponse}
+          reviewResponse={reviewResponse}
+          originalReviewResponse={originalReviewResponse}
+        >
+          {isActiveReviewResponse && decisionExists && <UpdateIcon onClick={showModal} />}
```

Comment:
it seems this will always have an edit button ? (unless it's not a draft review)

Reply:
This would show if (all is true):
- Review is in progress
- Reviewable by consolidator (isActiveReviewResponse)
- Consolidator has reviewed (decisionExists)

Do you see any case which is should/shouldn't display?

Reply:
> Consolidator has made decision (decisionExists)
I was thinking, the parent element should only show if decision exists

> Review is in progress
> Reviewable by consolidator (isActiveReviewResponse)
True, need to check isActiveReviewResponse

no need for change